README.md

```diff
@@ -118,6 +118,19 @@ Then configure Agent templates,
 assigning them labels that you can use so your jobs select the appropriate template,
 and set the docker container to be run with whatever container settings you require.
 
+### Running self-made local Docker images
+
+By default the Jenkins Docker plugin will try to download ('pull') the latest version of the image. This will fail with custom images that are not on Docker Hub. You will see logs like:
```

Comment:
```suggestion
By default, the Jenkins Docker plugin will try to download ('pull') the latest version of the image. This will fail with custom images that are not on Docker Hub. You will see logs like:
```